optimizely/client/client.go

```diff
@@ -133,6 +135,115 @@ func (o *OptimizelyClient) GetEnabledFeatures(userContext entities.UserContext)
 	return enabledFeatures, nil
 }
 
+// GetFeatureVariableString returns string feature variable value
+func (o *OptimizelyClient) GetFeatureVariableString(featureKey string, variableKey string, userContext entities.UserContext) (value string, err error) {
+	val, err := o.getFeatureVariable("", featureKey, variableKey, userContext)
+	if err == nil {
+		switch val.(type) {
+		case string:
+			return val.(string), err
+		default:
+			break
+		}
+	}
+	return "", err
+}
+
+func (o *OptimizelyClient) getFeatureVariable(valueType interface{}, featureKey string, variableKey string, userContext entities.UserContext) (value interface{}, err error) {
+	if !o.isValid {
```

Comment:
I feel this check, along with the project config check below can likely be collapsed into a `isValid` method that takes in the method name so it can print it out if not valid